pkgs/applications/networking/gns3/server.nix

```diff
@@ -24,7 +24,8 @@ in python.pkgs.buildPythonPackage {
 
   postPatch = ''
     substituteInPlace requirements.txt \
-      --replace "aiohttp==3.6.2" "aiohttp>=3.6.2"
+      --replace "aiohttp==3.6.2" "aiohttp>=3.6.2" \
+      --replace "py-cpuinfo==7.0.0" "py-cpuinfo>=7.0.0"
```

Comment:
https://github.com/GNS3/gns3-server/pull/1910 was merged.

Please add a comment that it should be removed with the next release.

Reply:
Hi, I have added line:
# postPatch replacements should be removed on the release.

does it look correct now?
I could not find any style guide for comments in nixpkgs